src/leaderboard.js

```diff
@@ -0,0 +1,12 @@
+import './style.css';
+
+export default function currentLeaderboard(result) {
```

Comment:
- [ ] Please avoid using the `function` keyword, instead use `arrow functions`, this is part of the module to make you a master in ES6 syntax 😄